docs/homepage.md

```diff
@@ -0,0 +1,45 @@
+# Homepage
+
+## Popular Links
+
+Popular links are the links that are present on GOV.UK under _'Popular on GOV.UK'_.
+These links are maintained through this Mainstream Publisher app.
+
+### How to edit popular links
+
+Popular links can be accessed and edited via the 'Homepage' link on Signon, for this we need the 'homepage_editor' permission inside the Mainstream Publisher app.
```

Comment:
Drive-by comment (found this PR via https://github.com/alphagov/govuk-developer-docs/pull/4763): why was the decision made to expose the 'homepage' functionality of Mainstream Publisher in Signon itself (instead of relying on users to navigate to the functionality via clicking "Publisher")?

It seems a bit of a confusing gotcha to add, unless I'm missing some clear rationale :/

Reply:
@ChrisBAshton This is a product decision about trialing the Signon links as more task-based than app-based. You can chat with our product owner (Emma Carter) if you'd like to learn more.

Reply:
Ah, thanks for the context - I'll ask Emma 👍 in the meantime it may be useful to add "Why is this being added directly to the Signon landing page?" to the "Questions" section in this doc.

Reply:
Hey @ChrisBAshton , we have now updated the doc to add this question.

Reply:
Perfect - thanks @syed-ali-tw